zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/metric/ZkClientMonitor.java

```diff
@@ -190,6 +191,12 @@ public void decreaseOutstandingRequestGauge() {
     }
   }
 
+  public void increaseZnodeCompressCounter() {
```

Comment:
Actual compressed ZNodes count is the ideal metric that we want, but I personally don't think it's doable in ZkClient level metrics. For example, do we report the compressed node written by the other ZkClients? If not, then what's the purpose of reporting an "inaccurate" compressed ZNode count?

The current plan is to report if any Znode is compressed through the ZkClient, counter is kind of showing the volume in addition to a boolean (0 or 1), so I pick up a counter metric. As for understanding the total compressed node, we need to leverage some scanning tools (too costly to be included in any online Helix instances). But this would be out of the scope of this PR.

After all, the metric is about how many new compressed znode are written. That's why it is called "CompressedZnodeWriteCounter" instead of "CompressedZnodeCounter".

Reply:
Thanks for the explanation. This makes sense then.